### ISSUE TITLE
Primary currency setting not working

### ISSUE THREAD

**Issue by omnat** [MetaMask/metamask-mobile#1311]
My primary currency is ETH:

<img src="https://user-images.githubusercontent.com/44038140/73506203-bab18c80-4389-11ea-90cd-d1ebac6af28c.png" width="250" height="400">

**But**, it isn't reflected in my accounts nor on transactions:

<img src="https://user-images.githubusercontent.com/44038140/73506370-33184d80-438a-11ea-837b-ab0842dd9692.jpg" width="250" height="400">

**Comment by estebanmino** [comment]
@omnat this is not a bug since we didn't add that behavior on these screens. Should we revisit this for the new send flow instead?

**Comment by omnat** [comment]
Good idea. Let's add this behavior of responding to primary currency settings in new transaction confirmations.
